.github/workflows/nightly-jdk11.yml

```diff
@@ -0,0 +1,43 @@
+name: Nightly jkd11
+
+on:
+  schedule:
+    - cron:  '0 6 * * *'
+
+jobs:
+  compile-and-test:
+    name: Compile and test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up JDK 11
```

Comment:
I think we can squash all the nightly in one single file and have separate jobs depending on the JDK, or even a single job switching JDK in the middle.

Reply:
(to at least re-use the scheduling and mailing boilerplate)